gsa/src/web/pages/reports/detailscontent.js

```diff
@@ -499,7 +499,7 @@ const PageContent = ({
                     onFilterDecreaseMinQoDClick={onFilterDecreaseMinQoDClick}
                     onFilterRemoveSeverityClick={onFilterRemoveSeverityClick}
                     onFilterEditClick={onFilterEditClick}
-                    onFilterResetClick={onFilterResetClick}
+                    onFilterRemoveClick={onFilterRemoveClick}
                     onInteraction={onInteraction}
```

Comment:
Why not keep both options? Removing the reset-option makes it impossible to easily apply the default filter

Reply:
I've only fixed the current behavior of the displayed box. If you think we should add another box when no results are available to actually reset the filter to the detauls a new PR is always welcome :smile: